.gitmodules

```diff
@@ -26,6 +26,6 @@
 [submodule "node-red-contrib-agile-solid"]
 	path = node-red-contrib-agile-solid
 	url = https://github.com/Agile-IoT/node-red-contrib-agile-solid.git
-[submodule "node-red-contrib-thingspeak42"]
-	path = node-red-contrib-thingspeak42
-	url = https://github.com/clough42/node-red-contrib-thingspeak42.git
+[submodule "node-red-contrib-agile-thingspeak"]
```

Comment:
It is actually there. I was also surprised to see it, since GitHub search is not showing it. But then, browsing through the repose from the GitHub repo list, I've found it. 
I suppose their search is going from off-line data build each night, or something similar. Or maybe they avoid showing forks without modifications.